Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -21,10 +21,11 @@
 using ISavable = Flow.Launcher.Plugin.ISavable;
 using System.IO;
 using System.Collections.Specialized;
+using CommunityToolkit.Mvvm.Input;
 
 namespace Flow.Launcher.ViewModel
 {
-    public class MainViewModel : BaseModel, ISavable
+    public partial class MainViewModel : BaseModel, ISavable
```

Comment:
@taooceros why is this changed to partial class?

Reply:
Because I would like to use the mvvm toolkit to simplify the code of creating RelayCommand, which requires partial to generate additional code.